### PR TITLE
[JBTM-2643] "number-of-application-rollbacks" statistic incremented twice

### PR DESCRIPTION
**Pull request by TomasHofman**
…during single rollback

!BLACKTIE !XTS !QA_JTA !QA_JTS_JACORB !QA_JTS_JDKORB !QA_JTS_IBMORB

https://issues.jboss.org/browse/JBTM-2643

Upstream for https://github.com/jbosstm/narayana/pull/991 (5.2 branch)

### CODE REVIEW

File: ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/TxStatsUnitTest.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
 import java.io.PrintWriter;
 import java.io.StringWriter;
 
+import org.junit.Ignore;
```
Comment (mmusgrov) [inline ArjunaCore/arjuna/tests/classes/com/hp/mwtests/ts/arjuna/atomicaction/TxStatsUnitTest.java]: This import is not used

**Comment by mmusgrov** [comment]
Thanks Thomas for this contribution.

**Comment by TomasHofman** [comment]
@tomjenkinson I selected Corporate, as I work for Red Hat - is that intended for outside contributors only?

**Comment by TomasHofman** [comment]
@tomjenkinson is there something I need to do to get this into 5.2 (for EAP 7)?

**Comment by TomasHofman** [comment]
I see, it's actually part of https://issues.jboss.org/browse/JBEAP-2456, linked as related issue. That JBEAP contains other unrelated issues, which may not going to be fixed so I want to extract this into separate JBEAP.

**Pull request merged**